### PR TITLE
[de/lua] First version of german translation of Lua

### PR DESCRIPTION
**Pull request by Git-Jiro**

### CODE REVIEW

File: de-de/lua-de.html.markdown [hunk 1]
```diff
@@ -0,0 +1,425 @@
+---
+language: Lua
+contributors:
+    - ["Tyler Neylon", "http://tylerneylon.com/"]
+translators:
+    - ["Martin Schimandl", "https://github.com/Git-Jiro"]
+filename: learnlua-de.lua
```
Comment (vendethiel) [inline de-de/lua-de.html.markdown]: please add a  `lang: de-de` field (and sorry for the delay) :).

Comment (Git-Jiro) [inline de-de/lua-de.html.markdown]: Fixed ;-)
Don't worry about the delay.

**Comment by vendethiel** [comment]
cool, thanks!

**Comment by vendethiel** [comment]
:cake:

**Pull request merged**